### PR TITLE
Re-scope container_puppet_config

### PR DESCRIPTION
**Pull request by kajinamit**
The container_puppet_config module was used to run puppet to generate config files and update config hash. However in EDPM we no longer use puppet but use this module to update config hash.

This reduces scope of the module by removing the implementation to run puppet because that part is no longer used.

Note that this change drops _exists method which is just a wrapper of os.path.exists .

### CODE REVIEW

**Comment by rebtoor** [comment]
@kajinamit please rebase this patch to main.

**Comment by raukadah** [comment]
recheck

**Comment by raukadah** [comment]
Logs from the last run: https://logserver.rdoproject.org/22/22/068bdddf003aab6b497976abfa2f63e39d17d316/github-check/edpm-ansible-github-rdo-integration-centos-8-crc-singlenode-centos-9-external-compute/d6e81b1/logs/controller/pod/deploy-external-dataplane-compute-q54gj-logs.txt .

**Comment by rebtoor** [comment]
@kajinamit i see that you're still committing patches to this PR. If it's not ready could you convert it to draft? I was going to merge it yesterday... :D

**Review by rebtoor — APPROVED** [review]

File: edpm_ansible/ansible_plugins/modules/container_config_hash.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    description:
+      - Config volume prefix
+    type: str
+    default: '/var/lib/config-data'
+  debug:
+    description:
+      - Enable debug
+    type: bool
+    default: False
+  step:
```
Comment (bogdando) [inline edpm_ansible/ansible_plugins/modules/container_config_hash.py:61]: this looks no longer used

**Review by bogdando — COMMENTED** [review]
have you tried ``git mv -f edpm_ansible/ansible_plugins/modules/container_config_hash.py edpm_ansible/ansible_plugins/modules/container_config_hash.py`` ? that would show the real diff instead of representing this script as a new file

**Pull request merged**